packages/devtools_app/lib/src/memory/flutter/memory_heap_tree_view.dart

```diff
@@ -423,9 +425,9 @@ class HeapTreeViewState extends State<HeapTree> with AutoDisposeMixin {
   }
 
   Future<void> doGroupBy() async {
-    controller.theGraph.computeInstancesForClasses();
-    controller.theGraph.computeRawGroups();
-    controller.theGraph.computeFilteredGroups();
+    controller.heapGraph.computeInstancesForClasses();
```

Comment:
tiny nit: this would be a nice case for method cascades.

Reply:
Done.